src/lib.rs

```diff
@@ -52,8 +52,17 @@ pub fn run(args: &clap::ArgMatches, current_dir: &PathBuf) -> Result<Vec<Warning
 
         let mut lines = get_line_entries(&fe, strs);
 
+        // run fixers & write results to file
         let mut result = checks::run(&lines, &skip_checks);
         if is_fix && fixes::run(&mut result, &mut lines) > 0 {
+            // create backup copy unless user specifies not to
+            let should_backup = !args.is_present("no-backup");
+            if is_fix && should_backup {
```

Comment:
`is_fix` seems extra, because code already in `if` snippet :slightly_smiling_face: 
```suggestion
            if should_backup {
```